README.md

```diff
@@ -118,6 +118,10 @@ To run the project, execute the following command:
 - GitHub: [@zewdu444](https://github.com/zewdu444)
 - Twitter: [@zewdu444](https://twitter.com/zewdu444)
 - LinkedIn: [zewdu-erkyhun](https://www.linkedin.com/in/zewdu-erkyhun-081378b3/)
+
```

Comment:
-[ ] OPTIONAL
Kindly remember to add appropriate content under all blank sections/headings.

Reply:
@rloterh thank you for the feedback we will correct it.